tools/dbTools.py

```diff
@@ -169,7 +139,7 @@ def update_full_article(self, article):
             WHERE id=%s;
             """,
             (article['title'], article['uri'], article['pdf'],
-             article['hash'], article.get('authors', ''), article['year'],
+             article['hash'], article.get('authors', ''), article.get('year'),
```

Comment:
why empty string instead of null for authors?

Reply:
That's wrong. Will fix 👍